addons/binding/org.openhab.binding.snapcast/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="snapcast"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="serverStreams">
+		<item-type>String</item-type>
+		<label>Streams</label>
+		<description>List of all available streams</description>
+		<state readOnly="true" />
+	</channel-type>
+	
+	<channel-type id="serverStreamsPlaying">
+		<item-type>String</item-type>
+		<label>Streams Playing</label>
+		<description>List of all playing streams</description>
+		<state readOnly="true" />
+	</channel-type>
+	
+	<channel-type id="serverStreamsIdle">
+		<item-type>String</item-type>
+		<label>Streams Idle</label>
+		<description>List of all idle streams</description>
+		<state readOnly="true" />
+	</channel-type>
+	
+	<channel-type id="clientName">
```

Comment:
What is the use case for this client-name? Does it make sense that it is read/write.